docs/site/DEVELOPING.md

```diff
@@ -587,3 +588,35 @@ This is why we have two sets of `tsconfig` files:
 - At monorepo root, there is `tsconfig.json` used by VS Code.
 - Inside each package, there is `tsconfig.build.json` used by `npm run build`
   command.
+
+## Renovate bot
+
+In loopback-next, we use package-lock files to speed up `npm install` times and
+[Renovate bot](http://renovatebot.com) to keep our lock files up to date.
+
+The bot is configured to maintain a special issue called
+`Update Dependencies (Renovate Bot)` where it lists all pull requests in
```

Comment:
Can we link to the issue? I assume it's https://github.com/strongloop/loopback-next/issues/3042?

Reply:
Done :)